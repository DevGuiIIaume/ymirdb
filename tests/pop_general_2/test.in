set a 1
set b a
set c 3
set d c
set e 5
list entries
push e a c
forward a
forward b
forward c
forward d
forward e
backward a
backward b
backward c
backward d
backward e
pop b
list entries
forward b
backward b
forward a
backward a
pop e
forward e
backward e
forward c
backward c
list entries
pop e
forward e
backward e
forward a
backward a
list entries
bye
