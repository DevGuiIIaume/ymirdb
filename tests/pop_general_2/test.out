> ok

> ok

> ok

> ok

> ok

> e [5]
d [c]
c [3]
b [a]
a [1]

> ok

> nil

> a

> nil

> c

> a, c

> b, e

> nil

> d, e

> nil

> nil

> a

> e [c a 5]
d [c]
c [3]
b []
a [1]

> nil

> nil

> nil

> e

> c

> a

> nil

> nil

> d

> e [a 5]
d [c]
c [3]
b []
a [1]

> a

> nil

> nil

> nil

> nil

> e [5]
d [c]
c [3]
b []
a [1]

> bye
